deploy/manifests/virtual-kubelet/base/deployment.yaml

```diff
@@ -103,10 +104,29 @@ spec:
           readOnly: true
         securityContext:
           privileged: true
+      - command:
+        - /bin/sh
+        - -c
+        - exec kube-proxy --oom-score-adj=-998 --bind-address=127.0.0.1 --v=2
+        image: k8s.gcr.io/kube-proxy:v1.18.3
+        imagePullPolicy: IfNotPresent
+        name: kube-proxy
+        resources:
+          requests:
+            cpu: 100m
+        securityContext:
+          privileged: true
+        terminationMessagePath: /dev/termination-log
+        terminationMessagePolicy: File
+        volumeMounts:
+        - mountPath: /run/xtables.lock
+          name: xtables-lock
+        - mountPath: /lib/modules
+          name: lib-modules
+          readOnly: true
       dnsPolicy: ClusterFirst
       restartPolicy: Always
       serviceAccountName: virtual-kubelet
-      hostNetwork: true
```

Comment:
I just realized this might be an issue?  Maybe?  How does this change interact with being able to reach the metadata service for AWS and GCE?

Reply:
That still works:

GCE:

    $ kubectl -n kube-system exec -ti virtual-kubelet-84584bd8fc-4zx8q -c virtual-kubelet -- bash
    bash-5.0# curl -H "Metadata-Flavor: Google" http://169.254.169.254/computeMetadata/v1/instance/name; echo
    gke-vilmos-node-pool-vilmos-2ae4f1eb-zfj1

AWS:

    $ kubectl -n kube-system exec -ti virtual-kubelet-776f968878-hbddt -c virtual-kubelet -- bash
    bash-5.0# curl http://169.254.169.254/latest/meta-data/instance-id; echo
    i-017c0ab709a2201ee